crates/sim2h/src/naive_sharding.rs

```diff
@@ -79,13 +81,17 @@ mod tests {
 
     #[test]
     fn it_should_safely_distribute_data() {
+        println!("starting test");
+
         let thread_cont = std::sync::Arc::new(std::sync::Mutex::new(true));
         let mut hash_threads = Vec::new();
 
         let (id_send, id_recv) = crossbeam_channel::bounded::<Location>(20);
         let (addr_send, addr_recv) = crossbeam_channel::bounded::<Location>(100);
 
         for _ in 0..8 {
+            println!("starting hash thread");
```

Comment:
Would be easy to switching to a `holochain_logger` `debug!` statement, as well as ones below it. (Yes you need that annoying `enable_logging_for_test` hack).